harvest/utils.py

```diff
@@ -1,13 +1,26 @@
+# Builtin Imports
 import re
+import sys
 import time
 import random
+import logging
 import datetime as dt
 from enum import IntEnum, auto
 
+# External Imports
 import pytz
 import tzlocal
 import pandas as pd
 
+formatter = logging.Formatter("%(asctime)s : %(name)s : %(levelname)s : %(message)s")
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s : %(name)s : %(levelname)s : %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+    handlers=[logging.FileHandler("harvest.log"), logging.StreamHandler(sys.stdout)],
+)
+debugger = logging.getLogger("harvest")
```

Comment:
Creates a logger callend harvest and outputs to harvest.log and standard out instead of standard error.